spec/models/classrooms_teacher_spec.rb

```diff
@@ -6,6 +6,23 @@
   let(:classrooms_teacher_with_null_classroom_id) { build(:classrooms_teacher, classroom_id: nil) }
   let(:classrooms_teacher) { build(:classrooms_teacher) }
 
+  it { should belong_to(:user) }
+  it { should belong_to(:classroom) }
+
+  it { is_expected.to callback(:delete_classroom_minis_cache_for_each_teacher_of_this_classroom).after(:create) }
+  it { is_expected.to callback(:reset_lessons_cache_for_teacher).after(:create) }
+  it { is_expected.to callback(:delete_classroom_minis_cache_for_each_teacher_of_this_classroom).before(:destroy) }
+  it { is_expected.to callback(:reset_lessons_cache_for_teacher).before(:destroy) }
+  it { is_expected.to callback(:trigger_analytics_events_for_classroom_creation).after(:commit).on(:create) }
+
+  describe 'teacher' do
+    let(:teacher) { create(:classrooms_teacher) }
```

Comment:
Can we call this `classrooms_teacher` for better clarity?

Reply:
yes